AIDevGallery/Controls/ModelPicker/ModelPickerViews/OnnxPickerView.xaml.cs

```diff
@@ -143,6 +143,16 @@ private void ModelSelectionItemsView_SelectionChanged(ItemsView sender, ItemsVie
         if (sender.SelectedItem is AvailableModel model)
         {
             OnSelectedModelChanged(this, model.ModelDetails);
+            DownloadModelSelectionItemsView.Select(-1);
+        }
+    }
+
+    private void DownloadModelSelectionItemsView_SelectionChanged(ItemsView sender, ItemsViewSelectionChangedEventArgs args)
+    {
+        if (sender.SelectedItem is DownloadableModel model)
+        {
+            OnSelectedModelChanged(this, model.GetModelDetails());
+            ModelSelectionItemsView.Select(-1);
```

Comment:
```suggestion
        {
            OnSelectedModelChanged(this, model.ModelDetails);
            DownloadModelSelectionItemsView.DeselectAll();
        }
    }

    private void DownloadModelSelectionItemsView_SelectionChanged(ItemsView sender, ItemsViewSelectionChangedEventArgs args)
    {
        if (sender.SelectedItem is DownloadableModel model)
        {
            OnSelectedModelChanged(this, model.GetModelDetails());
            ModelSelectionItemsView.DeselectAll();
```

nit: how about use this unified function to make it easier to understand what we are trying to do